crates/red_knot_vendored/vendor/typeshed/stdlib/red_knot.pyi

```diff
@@ -0,0 +1,34 @@
+from typing import TypeVarTuple, _SpecialForm
+
+
+# Special operations
+class TypeOf[T]: ...
+
+def assert_true(x: object) -> None: ...
+
+
+# Type extensions
+Ts = TypeVarTuple("Ts")
```

Comment:
I think this variable is unused! The PEP-695 type parameters you're using in the class and function definitions below all create their own local-to-the-scope variables